app/Http/Controllers/Auth/FirstVisitController.php

```diff
@@ -41,9 +45,18 @@ public function showFirstVisitManagerForm()
      */
     public function finishManagerRegistration(Request $request)
     {
-        $data = $request->all();
-        $validator = RegistrationValidator::finalizeManagerValidator($data);
-        $validator->validate();
+        $data = $request->validate([
+            'department' => 'required|integer',
+            'gov_email' => [
+                'nullable',
+                // gov_email is required unless department is set to 0 (Not in Government)
+                'required_unless:department,0',
+                'email:rfc',
+                'max:191',
+                // gov_email may match existing email for this user, must be unique if changed.
+                Rule::unique('users', 'email')->ignore($request->user()->id)
```

Comment:
Getting an error when registering and trying to use a `gov_email` that already exists. Pretty sure it's happening because the `user()` trying to register doesn't exist yet. 
![image](https://user-images.githubusercontent.com/22059495/75277420-f8ad9f00-57d5-11ea-8da8-8ca6d412b958.png)

Reply:
Fixed this. See the rules on RegistrationValidator vs. FirstVisitController.